app/controllers/subscriptions_controller.rb

```diff
@@ -1,12 +1,10 @@
 class SubscriptionsController < AuthenticatedController
-  include ProjectScoped
-
   def create
     subscription = Subscription.new(subscription_params)
     subscription.user = current_user
     subscription.save
 
-    redirect_back fallback_location: project_path(current_project), notice: 'Subscribed!'
+    redirect_back fallback_location: params[:fallback_path], notice: 'Subscribed!'
```

Comment:
let's use a sensible default (as before) instead of  user-controllable parameter.

`redirect_back` should already work in most normal cases, this is in case the browser didn't send a Referer header.

Reply:
Fixed.